build-logic/performance-testing/src/main/kotlin/gradlebuild/performance/tasks/BuildCommitDistribution.kt

```diff
@@ -71,27 +69,33 @@ abstract class BuildCommitDistribution @Inject internal constructor(
     @get:Optional
     abstract val commitBaseline: Property<String>
 
-    @get:OutputDirectory
-    abstract val commitDistributionHome: DirectoryProperty
+    @get:OutputFile
+    abstract val commitDistribution: RegularFileProperty
 
     @get:OutputFile
     abstract val commitDistributionToolingApiJar: RegularFileProperty
 
     init {
         onlyIf { commitBaseline.getOrElse("").matches(commitVersionRegex) }
```

Comment:
The more I think about this the more it makes sense to me to enable caching for this task. Then we don't need to do the whole TC copying around and local builds also would benefit from commit distributions built on CI. Moreover, we could even add a `buildCommitDistribution` build in the QuickFeedbackLinux stage which would prime the cache.

Reply:
Cacheable now.